### DISCUSSION TITLE
R.I.P Old Method?

### DISCUSSION THREAD

**Discussion by mmsiismm** [TheLastBen/fast-stable-diffusion#1117] [Q&A] [answered]
is the old method gone, or is there any old method link that works?
if not, does anyone know of a way to make the new method look as close as possible to the old method?
for me it was much easier to get good results in the old method  :'(

**Comment by TheLastBen** [comment]
set the learning a bit higher

**Reply by mmsiismm** [reply]
I may try this in the future

**Comment by ewsnebulam** [comment]
im not programmers but old method its just some lines like:
--Session_dir="+SESSION_DIR+" \
--class_data_dir="+CLASS_DIR+" \
--class_prompt="+CPT+" \
--with_prior_preservation --prior_loss_weight=1.0 \
--num_class_images="+str(num)+" \
maybe also
off Textenc?.....

**Reply by mmsiismm** [reply]
Thanks for the answer, it may be useful in the future

**Comment by TheLastBen** [comment, marked as answer]
use this from dotcsv https://colab.research.google.com/drive/1-HIbslQd7Ei_mAt25ipqSUMvbe3POm98?usp=sharing
https://www.youtube.com/watch?v=rgKBjRLvjLs

**Reply by mmsiismm** [reply]
Thank you!
